debian/rules

```diff
@@ -33,20 +33,6 @@ ifeq ($(DEB_HOST_ARCH_OS), linux)
 	GS_CONFIGURE_FLAGS += -Dfwupd=true
 endif
 
-# EOS-specific configuration
-GS_CONFIGURE_FLAGS += \
-	-Deos_updater=true \
-	-Dexternal_appstream=true \
-	-Dfwupd=false \
-	-Dgudev=false \
-	-Dhardcoded_curated=false \
-	-Dmalcontent=true \
-	-Dmogwai=true \
-	-Dpackagekit=false \
-	-Dsoup2=true \
-	-Dhardcoded_foss_webapps=false \
-	-Dhardcoded_proprietary_webapps=false
```

Comment:
I don't think it's correct to drop these flags. Higher up in the file, packagekit support (for example) is enabled, which we do not want.

Some of them may be redundant with the code-branch change to `meson_options.txt`, so it could perhaps be slimmed down.